src/amcrest/event.py

```diff
@@ -144,6 +144,11 @@ def is_motion_detected(self):
             return False
         return True
 
+    @property
+    def is_alarm_triggered(self):
+        event = self.event_channels_happened('AlarmLocal')
+        return bool('channels' in event)
```

Comment:
Why not just `return 'channels' in event`? The result of that expression would already be a `bool`.

Reply:
Ok as is.